cfgov/unprocessed/apps/teachers-digital-platform/img/icon-1.svg

```diff
@@ -1,9 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
If applicable, we have a `class="cf-icon-svg"` that does some sizing for SVGs. You can see it on the SVGs on https://cfpb.github.io/design-system/foundation/iconography if you inspect the source there.